src/operator/src/insert.rs

```diff
@@ -91,6 +94,20 @@ impl AutoCreateTableType {
     }
 }
 
+/// Split insert requests into normal and instant requests.
+///
+/// Where instant requests are requests with ttl=instant,
+/// and normal requests are requests with ttl set to other values.
+///
+/// This is used to split requests for different processing.
+#[derive(Clone)]
+pub struct InstantInsertRequests {
```

Comment:
The struct also has normal requests, maybe we should and normal to the struct name.